compare-rav1e.sh

```diff
@@ -3,11 +3,12 @@
 input=${1}
 cpu=${2}
 limit=${3}
+verbose=${4}
```

Comment:
This would need proper arg parsing. right now it breaks old scripts and doesn't work as intended since you do not shift by 4 below